src/libstd/thread/mod.rs

```diff
@@ -1052,6 +1053,30 @@ impl<T> JoinInner<T> {
 /// }).unwrap();
 /// ```
 ///
+/// Child being detached and outliving its parent:
+///
+/// ```
```

Comment:
This should be marked `no_run` so we compile the code but don't run it during tests (we don't want this test to take a full second)

see also: https://doc.rust-lang.org/nightly/book/first-edition/documentation.html#running-documentation-tests